src/main/java/uk/gov/companieshouse/insolvency/data/controller/RequestContext.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.companieshouse.insolvency.data.controller;
+
+public class RequestContext {
+
+    public static final String CONTEXT_ID = "contextId";
+
```

Comment:
Not sure we need this class, my view is we should pass contextId same as companyNumber or requestBody into service method